lgn-provers/src/params/checksum.txt

```diff
@@ -0,0 +1,2 @@
+1a0293f8e580a06f6304f7c5e5dd60a0f7f375abf1b708a454e35a9e9eae086f  zkmr_params/STORAGE_QUERY2/public_params
```

Comment:
4. Also this might not work with hardcoding `zkmr_params/` at the beginning of the path, since some operators changed that value I believe (CB). Should we just take out the root part (which is common to all different params path and therefore useless) ?